README.md

```diff
@@ -17,16 +17,73 @@ standard implementation of lightweight Virtual Machines (VMs) that feel and
 perform like containers, but provide the workload isolation and security
 advantages of VMs.
 
+## License
+
+The code is licensed under an Apache 2.0 license.
```

Comment:
Nit: I think "the Apache 2.0 license" is better, "an Apache 2.0 license" sounds a little as if we were using a custom variant. You're the native speaker though :)

Reply:
This text was dragged verbatim from the runtime README, but I agree so updated.